common/Models/ExtensionAdaptiveCard.cs

```diff
@@ -20,11 +23,18 @@ public class ExtensionAdaptiveCard : IExtensionAdaptiveCard
 
     public string TemplateJson { get; private set; }
 
+    private readonly IThemeSelectorService _themeSelectorService;

```

Comment:
Is the Dashboars StyleCop different from the SetupFlow StyleCop?  I believe it wants all `readonly` fields to be above `non-readonly` fields, ya?

Reply:
They're the same. I think the reason this doesn't complain is that the one above it is public, and "public should come before private" is a rule that we actually ignore in our stylecop, but its existence probably leads to this being OK.